diff --git a/node_modules/@mantine/core/lib/core/factory/create-polymorphic-component.d.ts b/node_modules/@mantine/core/lib/core/factory/create-polymorphic-component.d.ts
index 3179005..4e63481 100644
--- a/node_modules/@mantine/core/lib/core/factory/create-polymorphic-component.d.ts
+++ b/node_modules/@mantine/core/lib/core/factory/create-polymorphic-component.d.ts
@@ -13,7 +13,7 @@ export type PolymorphicComponentProps<C, Props = {}> = C extends React.ElementTy
     component: React.ElementType;
     renderRoot?: (props: Record<string, any>) => any;
 };
-export declare function createPolymorphicComponent<ComponentDefaultType, Props, StaticComponents = Record<string, never>>(component: any): (<C = ComponentDefaultType>(props: PolymorphicComponentProps<C, Props>) => React.ReactElement) & Omit<import("react").FunctionComponent<(Props & ComponentProp<any> & Omit<Omit<any, "ref">, "component" | keyof Props> & {
+export declare function createPolymorphicComponent<ComponentDefaultType, Props, StaticComponents = Record<string, never>>(component: any): (<C = ComponentDefaultType>(props: PolymorphicComponentProps<C, Props> & Props) => React.ReactElement) & Omit<import("react").FunctionComponent<(Props & ComponentProp<any> & Omit<Omit<any, "ref">, "component" | keyof Props> & {
     ref?: any;
     renderRoot?: ((props: any) => any) | undefined;
 }) | (Props & {
