diff --git a/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.Z9uaEzPr.mjs b/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.Z9uaEzPr.mjs
index 8f409bd..8bddd6e 100644
--- a/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.Z9uaEzPr.mjs
+++ b/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.Z9uaEzPr.mjs
@@ -6444,7 +6444,7 @@ function PluginInspect(options = {}) {
   }
   function configureServer(server) {
     const config = server.config;
-    Object.values(server.environments).forEach((env) => {
+    Object.values(server.environments ?? {}).forEach((env) => {
       const envCtx = ctx.getEnvContext(env);
       const _invalidateModule = env.moduleGraph.invalidateModule;
       env.moduleGraph.invalidateModule = function(...args) {
