diff --git a/node_modules/vite-plugin-checker/dist/esm/checkers/stylelint/options.js b/node_modules/vite-plugin-checker/dist/esm/checkers/stylelint/options.js
index 1fa4fb8..4788fcb 100644
--- a/node_modules/vite-plugin-checker/dist/esm/checkers/stylelint/options.js
+++ b/node_modules/vite-plugin-checker/dist/esm/checkers/stylelint/options.js
@@ -5,6 +5,7 @@ const translateOptions = async (command) => {
   const result = meow({
     autoHelp: false,
     autoVersion: false,
+    importMeta: import.meta,
     help: `
     Usage: stylelint [input] [options]
     Input: Files(s), glob(s), or nothing to use stdin.
@@ -90,7 +91,7 @@ const translateOptions = async (command) => {
 	`,
     flags: {
       allowEmptyInput: {
-        alias: "aei",
+        shortFlag: "aei",
         type: "boolean"
       },
       cache: {
@@ -118,60 +119,60 @@ const translateOptions = async (command) => {
         type: "string"
       },
       disableDefaultIgnores: {
-        alias: "di",
+        shortFlag: "di",
         type: "boolean"
       },
       fix: {
         type: "boolean"
       },
       formatter: {
-        alias: "f",
+        shortFlag: "f",
         default: "string",
         type: "string"
       },
       help: {
-        alias: "h",
+        shortFlag: "h",
         type: "boolean"
       },
       ignoreDisables: {
-        alias: "id",
+        shortFlag: "id",
         type: "boolean"
       },
       ignorePath: {
-        alias: "i",
+        shortFlag: "i",
         type: "string",
         isMultiple: true
       },
       ignorePattern: {
-        alias: "ip",
+        shortFlag: "ip",
         type: "string",
         isMultiple: true
       },
       maxWarnings: {
-        alias: "mw",
+        shortFlag: "mw",
         type: "number"
       },
       outputFile: {
-        alias: "o",
+        shortFlag: "o",
         type: "string"
       },
       printConfig: {
         type: "boolean"
       },
       quiet: {
-        alias: "q",
+        shortFlag: "q",
         type: "boolean"
       },
       reportDescriptionlessDisables: {
-        alias: "rdd",
+        shortFlag: "rdd",
         type: "boolean"
       },
       reportInvalidScopeDisables: {
-        alias: "risd",
+        shortFlag: "risd",
         type: "boolean"
       },
       reportNeedlessDisables: {
-        alias: "rd",
+        shortFlag: "rd",
         type: "boolean"
       },
       stdin: {
@@ -181,11 +182,11 @@ const translateOptions = async (command) => {
         type: "string"
       },
       version: {
-        alias: "v",
+        shortFlag: "v",
         type: "boolean"
       },
       globbyOptions: {
-        alias: "go",
+        shortFlag: "go",
         type: "string"
       }
     },
diff --git a/node_modules/vite-plugin-checker/dist/esm/logger.js b/node_modules/vite-plugin-checker/dist/esm/logger.js
index 06bfe1b..a371d17 100644
--- a/node_modules/vite-plugin-checker/dist/esm/logger.js
+++ b/node_modules/vite-plugin-checker/dist/esm/logger.js
@@ -235,9 +235,13 @@ function normalizeStylelintDiagnostic(diagnostic) {
         return null;
     }
     const loc = lineColLocToBabelLoc(d);
+    let cssSource = ''
+    if (diagnostic._postcssResult && 'css' in diagnostic._postcssResult) {
+      cssSource = diagnostic._postcssResult.css
+    }
     const codeFrame = createFrame(
       // @ts-ignore
-      diagnostic._postcssResult.css ?? "",
+      cssSource,
       loc
     );
     return {
