diff --git a/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.lGstipX0.mjs b/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.lGstipX0.mjs
index e8947f7..736c1b5 100644
--- a/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.lGstipX0.mjs
+++ b/node_modules/vite-plugin-inspect/dist/shared/vite-plugin-inspect.lGstipX0.mjs
@@ -3711,7 +3711,7 @@ function PluginInspect(options = {}) {
   }
   function configureServer(server) {
     const config = server.config;
-    Object.values(server.environments).forEach((env) => {
+    Object.values(server.environments ?? {}).forEach((env) => {
       const envCtx = ctx.getEnvContext(env);
       const _invalidateModule = env.moduleGraph.invalidateModule;
       env.moduleGraph.invalidateModule = function(...args) {
